sh2lib/examples/http2_request/main/Kconfig.projbuild

```diff
@@ -0,0 +1,9 @@
+menu "Example Configuration"
+
+    config EXAMPLE_NGHTTP_TASK_STACK_SIZE
+        int "nghttp task stack size"
```

Comment:
The examples mention `HTTP2`, maybe this can be renamed to be `EXAMPLE_NGHTTP2_TASK_STACK_SIZE`?

Reply:
Fixed